pkg/webui/console/views/device-general-settings/application-server-form/index.js

```diff
@@ -163,13 +181,20 @@ const ApplicationServerForm = React.memo(props => {
     >
       {showResetNotification && <Notification content={messages.keysResetWarning} info small />}
       <Form.Field
-        autoFocus
+        required
         title={sharedMessages.skipCryptoTitle}
         name="skip_payload_crypto_override"
         description={sharedMessages.skipCryptoDescription}
-        component={Checkbox}
+        component={RadioButton.Group}
+        decode={decodeSkipPayloadCrypto}
+        horizontal
```

Comment:
```suggestion
```

Given the longer option names (see above), I think we should use the vertical layout after all.